src/web/src/app/space/service.ts

```diff
@@ -27,23 +124,22 @@ export class SpaceService {
 	constructor(
 		private http: HttpClient,
 		private router: Router,
+		private requestWatcher: RequestWatcherService,
+		private injector: Injector,
```

Comment:
router и injector не используются
```suggestion
		private requestWatcher: RequestWatcherService,
```

Reply:
Готово.